x-pack/plugins/ml/public/components/anomalies_table/anomalies_table.js

```diff
@@ -202,11 +203,39 @@ function getColumns(
     }
   }
 
-  columns.push({
-    field: 'jobId',
-    name: 'job ID',
-    sortable: true
-  });
+  if (examplesByJobId && Object.keys(examplesByJobId).length > 1) {
```

Comment:
I don't think this check is quite right. If more than 1 job is selected in the view (specifically the Anomaly Explorer), then the table should show always job ID I think.

Reply:
Yep - that one snuck by me - thank you!